### DISCUSSION TITLE
Wiki license

### DISCUSSION THREAD

**Discussion by Alxira5** [SuperTux/supertux#2905] [General]
Hello community, I created this topic to ask a small question regarding the wiki, which is about your license. If I'm not mistaken the old wiki used the [GFDL 1.2](https://web.archive.org/web/20221204023342/http://supertux.lethargik.org/wiki/Main_Page), which is a free license, but the new wiki does not have any type of license and could say that it is not free.

Wouldn't it be possible to change the wiki license or maybe give a way to reuse the content of the wiki?

**Comment by ghost** [comment]
I think it is CC BY SA 4.0, because all non-code components of SuperTux are licensed under CC BY SA 4.0

**Reply by Alxira5** [reply]
But I'm not really sure if it applied, because I can't find a page or footer that says so. I'm only sure that the old wiki was under GFDL 1.2.
